llvm/test/CodeGen/AMDGPU/call-args-inreg-no-sgpr-for-csrspill.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: not --crash llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 %s -o - 2>&1 | FileCheck -enable-var-scope %s
+
```

Comment:
Should also add a .xfail or something to the test name 

Also replace -o - with -filetype=null

Reply:
> Also replace -o - with -filetype=null

Is that what we prefer to use now?

Reply:
You aren't using stdout, so don't produce it